packages/sdk/client-services/src/packlets/services/service-host.ts

```diff
@@ -47,18 +48,27 @@ export class ClientServicesHost implements ClientServicesProvider {
   private readonly _modelFactory: ModelFactory;
   private readonly _networkManager: NetworkManager;
   private _storage: Storage;
+  private _open = false;
+
+  readonly compatibilityMode: boolean;
 
   constructor({
     config,
     modelFactory = createDefaultModelFactory(),
     // TODO(burdon): Create ApolloLink abstraction (see Client).
     networkManager,
-    storage = createStorageObjects(config.get('runtime.client.storage', {})!).storage
+    storage = createStorageObjects(config.get('runtime.client.storage', {})!).storage,
+    compatibilityMode = false
   }: ClientServicesHostParams) {
     this._config = config;
     this._modelFactory = modelFactory;
     this._networkManager = networkManager;
     this._storage = storage;
+    this.compatibilityMode = compatibilityMode;
```

Comment:
I'd initialize service registry with systemService here in the constructor so they are always available

then `open` and `close` control only other services

Reply:
Service registry might need to become mutable for this